src/Confirm.js

```diff
@@ -4,9 +4,9 @@ var { Button, Modal } = require('react-bootstrap');
 var Confirm = React.createClass({
     propTypes: {
         body: React.PropTypes.any.isRequired,
-        buttonText: React.PropTypes.string,
-        cancelText: React.PropTypes.string,
-        confirmText: React.PropTypes.string,
+        buttonText: React.PropTypes.any,
```

Comment:
Can you change this to `React.PropTypes.node` instead of `any`. That way there's still a warning if someone passes a object or something that can't be rendered.